scipy/special/xsf/numpy.h

```diff
@@ -14,14 +14,37 @@
 
 #include <numpy/arrayobject.h>
 #include <numpy/npy_3kcompat.h>
+#include <numpy/npy_math.h>
 #include <numpy/ufuncobject.h>
 
 #include "dual.h"
-#include "sf_error.h"
+#include "error.h"
 #include "third_party/kokkos/mdspan.hpp"
 
+/* PyUFunc_getfperr gets bits for current floating point error (fpe) status codes so we
+ * can check for floating point errors and make proper calls to set_error in ufunc loops.
+ * Define a wrapper so it can be given C linkage within this C++ header. */
+extern "C" int wrap_PyUFunc_getfperr() { return PyUFunc_getfperr(); }
+
```

Comment:
I made the comment more accurate here, and also moved this out of the C++ namespaces since it has C-linkage. I didn't use the namespace prefix in this file anyway.